readme.md

```diff
@@ -20,13 +20,13 @@ Create a pages with product card using `flexbox`, `BEM` and `SCSS` based on [thi
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_product-cards/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_product-cards/report/html_report/)
+- [DEMO LINK](https://Miranna472.github.io/layout_product-cards/)
+- [TEST REPORT LINK](https://Miranna472.github.io/layout_product-cards/report/html_report/)
```

Comment:
Add these links to the PR description 
<img width="991" alt="image" src="https://github.com/user-attachments/assets/e6ae721c-3ceb-4f16-8d52-5961caec92c7">